src/lib/oauth2-issuer.ts

```diff
@@ -87,6 +87,7 @@ export class OAuth2Issuer extends EventEmitter {
     const timestamp = Math.floor(Date.now() / 1000);
 
     const header: Header = {
+      alg: arguments.length === 0 || signed ? getKeyAlg(key) : 'none',
```

Comment:
@poveden Could you please elaborate what led to this change?

Reply:
Sure! `tsc` complains that `JwtHeader.alg` is required:

```
src/lib/oauth2-issuer.ts:125:7 - error TS2741: Property 'alg' is missing in type 'Header' but required in type 'JwtHeader'.
```

So I added `alg` to `src/lib/types.ts` and moved the assignment from `jwt.SignOptions.algorithm` to here.